Formula/sundials.rb

```diff
@@ -1,8 +1,8 @@
 class Sundials < Formula
   desc "Nonlinear and differential/algebraic equations solver"
   homepage "https://computation.llnl.gov/casc/sundials/main.html"
-  url "https://computing.llnl.gov/projects/sundials/download/sundials-5.4.0.tar.gz"
-  sha256 "04d8a2ebe02cdaeef5a9e22ff7e3146bb563d8400f65772b6c7af80001413ffa"
+  url "https://computation.llnl.gov/projects/sundials/download/sundials-5.5.0.tar.gz"
```

Comment:
`https://computation.llnl.gov` redirects to `https://computing.llnl.gov` . Could we `computing.nnl` ?

Reply:
good callout, I also updated the homepage as well.